### ISSUE TITLE
Honeycomb not rendering in older browsers

### ISSUE THREAD

**Issue by ghost** [dribdat/dribdat#90]
The viz of the progress of the different project look wired in my browser (Safari).

**Comment by loleg** [comment]
This is due to lack of full CSS3 support. We should fallback to rendering a grid or even only the list view.

**Comment by loleg** [comment]
The ideal solution here would be to have CSS 2-specific rendering. I haven't revisited this in a while, but just a couple of bullet points on this: 

- The grid renders fine in Safari on an old iPhone 5 which is almost 10 years old, and I haven't heard about this complaint from other users.
- I'm revamping the interface of dribdat in the PWA https://github.com/dribdat/backboard project where we can do more for retro devices.
- The new search interface in the about page (which I want to soon make more prominent), profile and resources areas, have a simpler grid for project display. If there is interest, I will make it an option for the event grid, too.

In other words, if this is still an issue, please feel free to re-open!
